src/components/pages/Offers/Offer/OfferDetails/OfferDetails.jsx

```diff
@@ -132,20 +137,25 @@ const OfferDetails = ({
       <div className="sidebar-container">
         <div className="content">
           {offer ? (
-            <OfferEditionContainer
-              formValues={formValues}
-              isLoading={isLoading}
-              isUserAdmin={isUserAdmin}
-              offer={offer}
-              onSubmit={handleSubmitOffer}
-              setFormValues={setFormValues}
-              setIsLoading={setIsLoading}
-              setPreviewOfferType={setOfferType}
-              setShowThumbnailForm={setShowThumbnailForm}
-              showErrorNotification={showErrorNotification}
-              submitErrors={formErrors}
-              userEmail={userEmail}
-            />
+            <>
+              {(offerStatus === OFFER_STATUS.REJECTED || offerStatus === OFFER_STATUS.AWAITING) && (
```

Comment:
```suggestion
              const needsStatusInfosMessage = (offerStatus === OFFER_STATUS.REJECTED || offerStatus === OFFER_STATUS.AWAITING) 
              { needsStatusInfosMessage && (
```
essaie plutôt de mettre les statuts en question dans une liste pour faciliter sa mise à jour